src/main/java/domain/LadderResult.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class LadderResult {
+
+    private final Map<String, String> results;
+
+    public LadderResult(Map<String, String> results) {
+        this.results = results;
+    }
+
+    public List<String> getWinning(String player) {
+        if (player.equals("all")) {
+            return getAll();
+        }
```

Comment:
```suggestion
        if ("all".equals(player)) {
            return getAll();
        }
```

상수에서 equals를 호출하여 npe를 방지해주세요
그리고 player 보다는 playerName이 더 적절하지 않을까요?🤔

Reply:
all을 상수로 빼는것도 고려해주세요